dbms/src/Processors/Executors/PipelineExecutor.cpp

```diff
@@ -577,32 +577,45 @@ void PipelineExecutor::executeImpl(size_t num_threads)
     for (size_t i = 0; i < num_threads; ++i)
         executor_contexts.emplace_back(std::make_unique<ExecutorContext>());
 
+    auto thread_group = CurrentThread::getGroup();
+
```

Comment:
This part looks unrelated, should it be split to a separate commit?

Reply:
Oh, I have forgotten about that small change.
Can we keep it here?